pom.xml

```diff
@@ -116,6 +132,12 @@
             <version>2.31.33</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>ssooidc</artifactId>
+            <version>2.31.33</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I needed this to do basic testing directly against the AWS credentials available in my organization. It doesn't work for testing stuff related to the agent role assumption logic (seems like it should be possible with `AWS_CHAINED_ROLE`, but I couldn't quite get that working after spending a few minutes with it), and here I am not changing any of that logic anyway.

Reply:
Ok, I confirmed that this is required even after getting `AWS_ROLE` to work (problem was that I was trying to use the display name instead of the ARN for the role).